go.mod

```
@@ -0,0 +1,3 @@
+module urlshortener
```

так тоже можно, но как некоторый стандарт индустрии принято называть по адресу репозитория 
```suggestion
module github.com/StepanIT/URL-shortening-service
```